presenters/thread.rb

```diff
@@ -23,7 +23,7 @@ def initialize(thread, user, is_read, unread_count, is_endorsed)
     @is_endorsed = is_endorsed
   end
 
-  def to_hash with_responses=false, resp_skip=0, resp_limit=nil
+  def to_hash with_responses=false, resp_skip=0, resp_limit=nil, recursive=false
```

Comment:
Just to confirm: The previous behavior was recursive.  The new default is non-recursive.  So without any other changes to client code, this is not a backwards compatible change.  Have you confirmed that you caught all instances where this code might be called from edx-platform, and updated them to use the recursive argument properly?